### PR TITLE
Nt run optimize

### PR DESCRIPTION
**Pull request by smmathews-bw-boston**

### CODE REVIEW

File: roaringbitmap.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include "libpq/pqformat.h"
 
 /* must include "roaring.h" before redefine malloc functions */
+#define CROARING_ATOMIC_IMPL 3
```
Comment (acmcmurray) [inline roaringbitmap.h:26]: Is this just for the stuff in `portability.h`?

File: roaringbitmap.c [hunk 2]
```diff
@@ -2053,3 +2053,30 @@ rb_cardinality_final(PG_FUNCTION_ARGS) {
...
+PG_FUNCTION_INFO_V1(rb_or);
+Datum rb_or(rb_runoptimize);
+
+Datum
+rb_or(PG_FUNCTION_ARGS) {
+    bytea *serializedbytes = PG_GETARG_BYTEA_P(0);
+    roaring_bitmap_t *r;
+    bool optimized;
+
+    r = roaring_bitmap_portable_deserialize(VARDATA(serializedbytesin));
```
Comment (acmcmurray) [inline roaringbitmap.c:2067]: Should you be doing any null checks before running this? Like functions above?

**Review by acmcmurray — COMMENTED** [review]
My knowledge is not good enough to give this a 👍 but it looks sensible to me

**Comment by acmcmurray** [comment]
@bwrobc @bashtanov know C 🧠

**Comment by smmathews-bw-boston** [comment]
closing in favor of https://github.com/BrandwatchLtd/pg_roaringbitmap/pull/3, which isn't based on upgrading our roaring bitmap version

File: roaringbitmap.h (hunk 1 above)
Comment (smmathews-bw-boston) [reply roaringbitmap.h:26]: removed in https://github.com/BrandwatchLtd/pg_roaringbitmap/pull/3

File: roaringbitmap.c (hunk 2 above)
Comment (smmathews-bw-boston) [reply roaringbitmap.c:2067]: I don't think that's necessary. functions like rb_to_array seem to be able to avoid it entirely.

**Pull request closed without merging**